bun/starter/whatsapp-with-vonage/README.md

```diff
@@ -0,0 +1,177 @@
+| ------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| Required      | Yes                                                                                                                      |
+| Sample Value  | `P2...97`                                                                                                                |
+
+## VONAGE_WHATSAPP_NUMBER
+
+Vonage WhatsApp number to send messages from.
+
+| Question      | Answer                                                                                                                        |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| Required      | Yes                                                                                                                           |
+| Sample Value  | `1234567890`                                                                                                                 
+
+
+
+## VONAGE_ACCOUNT_SECRET
+
+ACCOUNT_SECRET to use the Vonage API.
+
+| Question      | Answer                                                                                                                   |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| Required      | Yes                                                                                                                      |
+| Sample Value  | `p524...97`                                                                                                                |
+
+## VONAGE_SIGNATURE_SECRET
+
+Secret to verify the webhooks sent by Vonage.
+
+| Question      | Answer                                                                                                         |
+| ------------- | -------------------------------------------------------------------------------------------------------------- |
+| Required      | Yes                                                                                                            |
+| Sample Value  | `ZACYi3...ZXGjsch`                                                                                                |
+
+## CONFIGURATION
+
+
+| Setting           | Value                    |
+| ----------------- | ------------------------ |
+| Runtime           | Bun (1.0)              |
+| Entrypoint        | `src/main.js`            |
+| Build Commands    | `Bun run src/main.js` |
+| Permissions       | `any`                    |
+| Timeout (Seconds) | 15                       |
+
+
+## ENVIRONMENT VARIABLES
+
+Following environment variables are required :
+
+`VONAGE_API_KEY`
+
+`VONAGE_ACCOUNT_SECRET`
+
+`VONAGE_SIGNATURE_SECRET`
+
+`VONAGE_WHATSAPP_NUMBER`
+
+`PORT`
+## Run Locally
```

Comment:
Please remove these additional sections and keep this consistent with the existing Whatsapp vonage templates